pkg/plugins/datasource_plugin.go

```diff
@@ -26,18 +26,17 @@ import (
 // DataSourcePlugin contains all metadata about a datasource plugin
 type DataSourcePlugin struct {
 	FrontendPluginBase
-	Annotations   bool              `json:"annotations"`
-	Metrics       bool              `json:"metrics"`
-	Alerting      bool              `json:"alerting"`
-	Explore       bool              `json:"explore"`
-	Table         bool              `json:"tables"`
-	HiddenQueries bool              `json:"hiddenQueries"`
-	Logs          bool              `json:"logs"`
-	QueryOptions  map[string]bool   `json:"queryOptions,omitempty"`
-	BuiltIn       bool              `json:"builtIn,omitempty"`
-	Mixed         bool              `json:"mixed,omitempty"`
-	Routes        []*AppPluginRoute `json:"routes"`
-	Streaming     bool              `json:"streaming"`
+	Annotations  bool              `json:"annotations"`
+	Metrics      bool              `json:"metrics"`
+	Alerting     bool              `json:"alerting"`
+	Explore      bool              `json:"explore"`
+	Table        bool              `json:"tables"`
+	Logs         bool              `json:"logs"`
+	QueryOptions map[string]bool   `json:"queryOptions,omitempty"`
+	BuiltIn      bool              `json:"builtIn,omitempty"`
+	Mixed        bool              `json:"mixed,omitempty"`
+	Routes       []*AppPluginRoute `json:"routes"`
+	Streaming    bool              `json:"streaming"`
```

Comment:
I *THINK* it was added as a flag for explore...  and shows the "live" button.

but I would rather we remove it and handle as per query metadata rather than per datasource